test/system/post_test.rb

```diff
@@ -38,26 +38,28 @@ def setup
 
     find('.tag-input').set('nature').native.send_keys(:return)
     find('.tag-input').set('mountains').native.send_keys(:return)
+    find('.tag-input').set('test:one').native.send_keys(:return)
 
     # Make sure that the 2 tags are added
-    page.assert_selector('.tags-list p.badge', :count => 2)
+    page.assert_selector('.tags-list p.badge', :count => 3)
   end
 
   test 'removing tags from the post' do
     visit '/wiki/organizers'
 
     find('a#tags-open').click()
 
-    # There should be 1 tag that shows up as a badge and 2 as a card
-    page.assert_selector('.tags-list .card-body', :count => 2)
-    page.assert_selector('.tags-list p.badge', :count => 1)
+    # There should be 3 tags that show up as a card
+    page.assert_selector('.tags-list .card-body', :count => 3)
+
+    find('.tags-list:first-of-type .card-body .ellipsis').click()
```

Comment:
@TildaDares this is the line which causes the system test to fail. I used `first-of-type` to select only the first tag but I'm not sure why it still fails.

Reply:
@Manasa2850 Have you tried using `first`? https://www.rubydoc.info/gems/capybara/Capybara%2FNode%2FFinders:first

Reply:
Thank you so much @TildaDares!! That worked :tada: